src/pages/iou/request/step/IOURequestStepScan/index.native.tsx

```diff
@@ -73,7 +54,7 @@ function IOURequestStepScan({
         // There's no way we can check for the BLOCKED status without requesting the permission first
         // https://github.com/zoontek/react-native-permissions/blob/a836e114ce3a180b2b23916292c79841a267d828/README.md?plain=1#L670
         CameraPermission.requestCameraPermission()
-            .then((status) => {
+            .then((status: string) => {
```

Comment:
1. I dont think that this `:string` is necessary because `CameraPermission` module is already in TS and `status` type can be inferred.
2. `.then` for sure is there if you look to `CameraPermission` module you can see that `requestCameraPermission` method is returning `Promise`

Reply:
I checkout to your branch and see the problem, if you look at `CameraPermission` module it doesn't follow TS guidelines for modules which have platform specific files (https://github.com/Expensify/App/blob/main/contributingGuides/TS_STYLE.md#file-organization) so my recommendation would be to adjust this module to match or guidelines